tests/ui/doc/doc.stderr

```diff
@@ -1,160 +1,160 @@
-error: you should put `foo_bar` between ticks in the documentation
+error: item in documentation missing backticks
```

Comment:
Could you add `// run-rustfix` in the first line of this and other `.rs` test file? That'll create a `.fixed` file and validate that the suggestion has been applied correctly :upside_down_face: